ADOPTERS.md

```diff
@@ -71,7 +71,7 @@
 - [The Zebra](https://www.thezebra.com)
 - [Tradeshift](https://tradeshift.com/)
 - [Transit](https://transit.app)
-- [Vernacular.ai](https://vernacular.ai/)
+- [Skit](https://skit.ai/)
```

Comment:
Can you move this up, as the list is in alphabetical order?

Reply:
Done.